docs/.vuepress/config.js

```diff
@@ -82,5 +82,9 @@ module.exports = {
   plugins: [
     '@vuepress/plugin-back-to-top',
     '@vuepress/plugin-medium-zoom',
+    '@vuepress/google-analytics',
+    {
+      'ga': 'UA-28106310-7'
```

Comment:
@blushi Can you double check the ID and make sure `docs.regen.network` is the only URL we are tracking?

Reply:
I added you to google analytics so you can double check and do additional setup if needed, I've just setup a new web data stream for https://docs.regen.network/